configuration/rules-dsl.md

```diff
@@ -207,6 +207,19 @@ Besides the implicitly available variables for items and commands/states, rules
 - Every rule that has at least one command event trigger, will have the variable `receivedCommand` available, which can be used inside the execution block.
 - Every rule that has at least one status change event trigger, will have the variable `previousState` available, which can be used inside the execution block.
 
+### Early returns
+
+Similar to Java, it is possible to return early from a rule, not executing the rest of the statements like this:
+
+```java
+if (Temperature.state > 20) {
+	return;
+}
+sendCommand(Heating, ON)
+```
+
+Caveat: Please note the semicolon after the return statement. If you leave it out, the next line might still get executed. 
```

Comment:
"might" doesn't create confidence in a determined openHAB behavior. In my opinion just leave it with the first sentence or add something as a short rationale like "which terminates the command without an additional argument." wdyt?